tests/ui/structs-enums/rec-align-u64.rs

```diff
@@ -77,7 +77,7 @@ mod m {
     }
 }
 
-#[cfg(target_family = "wasm")]
+#[cfg(all(target_family = "wasm", not(target_os = "emscripten")))]
```

Comment:
Suggestion: please add a brief remark for why this is special on `empscripten` (FIXME of unknown root cause is fine).